java/src/main/java/de/gdata/vaas/Vaas.java

```diff
@@ -63,61 +61,55 @@ public void disconnect() throws InterruptedException {
         }
     }
 
-    public CompletableFuture<VerdictResponse> forUrlAsync(URL url) throws Exception {
+    public CompletableFuture<VerdictResponse> forUrlAsync(URL url, HashMap<String, String> verdictRequestAttributes) throws Exception {
```

Comment:
General question: Do we expect a user of the SDK to always provide tenant info? I would assume no, in which case the API gets significantly more difficult to use.

Does it make sense to have new API function with the support of attributes instead of adding it to the existing functions? This would also make a breaking change a non-breaking change as no update on the users side is needed.